modules/nf-core/nacho/normalise/main.nf

```diff
@@ -0,0 +1,58 @@
+nextflow.enable.moduleBinaries = true
+
+process NACHO_NORMALISE {
+    tag "$sample_sheet"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container 'community.wave.seqera.io/library/r-dplyr_r-fs_r-ggplot2_r-nacho_pruned:033bc017f5f36b6d'
+
+    input:
+    path rcc_files, stageAs: "input/*"
+    path sample_sheet
```

Comment:
We might want a `meta` here?
https://nf-co.re/docs/guidelines/components/modules#types-of-meta-fields